src/docs/usr-manual/ch-intro.rst

```diff
@@ -195,36 +200,31 @@ application.
   for building matrices in particular formats, find the IJ interface relatively
   easy to use. See Chapter :ref:`ch-IJ` for details.
 
-.. _fig-ls-interface:
-
-.. figure:: figConcepIface.*
+.. figure:: figConcepIface.svg
    :align: center
+   :name: fig-ls-interface
 
-   Figure 1
-
-   Graphic illustrating the notion of conceptual linear system interfaces.
+   : Graphic illustrating the notion of conceptual linear system interfaces.
 
 Generally, a user should choose the most specific interface that matches their
-application, because this will allow them to use specialized and more efficient
-solvers and preconditioners without losing access to more general solvers.  For
-example, the second row of Figure :ref:`fig-ls-interface` is a set of linear
-solver algorithms.  Each linear solver group requires different information from
-the user through the conceptual interfaces.  So, the geometric multigrid
-algorithm (GMG) listed in the left-most box, for example, can only be used with
-the left-most conceptual interface.  On the other hand, the ILU algorithm in the
-right-most box may be used with any conceptual interface.  Matrix requirements
-for each solver and preconditioner are provided in Chapter :ref:`ch-Solvers` and
-in Chapter :ref:`ch-API`.  Your desired solver strategy may influence your
-choice of conceptual interface.  A typical user will select a single Krylov
-method and a single preconditioner to solve their system.
-
-The third row of Figure :ref:`fig-ls-interface` is a list of data layouts or
-matrix/vector storage schemes.  The relationship between linear solver and
-storage scheme is similar to that of the conceptual interface and linear solver.
-Note that some of the interfaces in hypre currently only support one
-matrix/vector storage scheme choice.  The conceptual interface, the desired
-solvers and preconditioners, and the matrix storage class must all be
-compatible.
+application, because this will allow them to use specialized and more efficient solvers
+and preconditioners without losing access to more general solvers.  For example, the
+second row of :numref:`fig-ls-interface` is a set of linear solver algorithms.  Each linear
```

Comment:
Hi @victorapm.  I think I like the new `:numref:` approach better for figures than the `:ref:` approach above in the paragraph starting with "An important decision to make ...".  I could easily be swayed to go the other way with this, but maybe we should leave that for a later discussion since it would require more changes to the manual (and carefully written short titles).  Could you change the above figure references that use `:ref:` and maybe also do a quick search for others to fix?  Change only figure references, not chapter references.

Reply:
Done!